localstack/services/events/provider_v2.py

```diff
@@ -0,0 +1,75 @@
+import logging
```

Comment:
ℹ️  At least once we make the new v2 officially available, I think it makes sense to move all the v1-related functionality into a `legacy` (or v1) sub-directory (which might require ext changes!). This makes it easier to distinguish old from new.

We should synchronize on that one. It's probably better to defer that change until my event filtering refactorings and fixes are merged.